recipe_scrapers/_schemaorg.py

```diff
@@ -33,16 +33,19 @@ def __init__(self, page_data):
                     return
                 elif in_context and "@graph" in item:
                     for graph_item in item.get("@graph", ""):
-                        in_graph = SCHEMA_ORG_HOST in graph_item.get("@context", "")
-                        if (
-                            in_graph
-                            and graph_item.get("@type", "").lower() in low_schema
-                        ):
+                        graph_item_type = graph_item.get("@type", "")
+                        if type(graph_item_type) != str:
```

Comment:
Just taking a quick look at this PR and will look deeper later.

Wouldn't we want to do something like `if not isinstance(graph_item_type, str)` since we also would want this check to apply to `str` subclasses. However, I realize in this context we'd never have a string subclass.

Reply:
Yeah, you are probably right. My apologies, I've been jumping between languages lately and my python is wearing off :smirk: